### ISSUE TITLE
Python 3.4 incompatibility

### ISSUE THREAD

**Issue by psychemedia** [pascal-schetelat/Slope#1]
Line 264 breaks on import to python 3.4

Needs to be:

print('fail')

**Comment by pascal-schetelat** [comment]
Did you see other Python 3 issue ?

**Comment by psychemedia** [comment]
I'm getting (the same) matplotlib error on the first two slope() examples in the demo notebook but not the third:

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-bc21aead65ec> in <module>()
----> 1 f = slope(data/1000,width =30,height= 12,kind='interval',font_size=20,color=color,savename=None,dpi=200)

/vagrant/notebooks/Slope-master/plotSlope.py in slope(data, kind, marker, color, title, font_family, font_size, width, height, ax, savename, dpi, wspace)
    227             elif marker :
    228                 ax.set_yticks(yPos_L.values)
--> 229                 ax.set_yticklabels(ylabelsL_str,fontproperties = font,ha='right')#ha='center')#,backgroundcolor='w')
    230             else :
    231                 plt.setp(ax.get_yticklabels(),visible=False)

/usr/lib/python3/dist-packages/matplotlib/axes.py in set_yticklabels(self, labels, fontdict, minor, **kwargs)
   2863         """
   2864         return self.yaxis.set_ticklabels(labels, fontdict,
-> 2865                                          minor=minor, **kwargs)
   2866 
   2867     def xaxis_date(self, tz=None):

/usr/lib/python3/dist-packages/matplotlib/axis.py in set_ticklabels(self, ticklabels, *args, **kwargs)
   1509         ret2 = []
   1510         for i, tick in enumerate(ticks):
-> 1511             if i < len(ticklabels):
   1512                 if tick.label1On:
   1513                     tick.label1.set_text(ticklabels[i])

TypeError: object of type 'map' has no len()
```

Not had chance to check/update my version of matplotlib...

The third slope() example plotted fine.

**Comment by pascal-schetelat** [comment]
Fixed it with 393483fe71d231cc7a48db502f329810ca096bb2

Wasn't due to Matplotlib, but o Python3 usage of iterable and mapping.
